Api/Helper/Csv.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Werkspot\BingAdsApiBundle\Api\Helper;
+
+class Csv
+{
+    const FILE_HEADERS = 10;
+    const COLUMN_HEADERS = 1;
+
+    /**
+     * @param array $lines
+     * @param bool $includingColumnHeaders
+     *
+     * @return array
+     */
+    public function removeHeaders(array $lines, $includingColumnHeaders = true)
+    {
+        $lines = array_values($lines);
+
+        $removeLines = ($includingColumnHeaders) ? (self::FILE_HEADERS + self::COLUMN_HEADERS) : self::FILE_HEADERS;
+        for ($i = 0; $i < ($removeLines); $i++) {
+            unset($lines[$i]);
+        }
+
+        $lines = array_values($lines);
+        return $lines;
+    }
+
+    /**
+     *
+     * @param array $lines
+     * @param int   $noOfLinesToRemove
+     *
+     * @return array
+     */
+    public function removeLastLines(array $lines, $noOfLinesToRemove = 1)
+    {
+        $totalLines = count($lines);
+        $removeFrom = $totalLines - $noOfLinesToRemove;
+
+        for( $i = $removeFrom; $i < $totalLines; $i++ ) {
+            unset($lines[$i]);
+        }
+
+        $lines = array_values($lines);
+        return $lines;
+    }
+
+    /**
+     * @param array $lines
+     * @param string $separator
+     * @param string $enclosure
+     *
+     * @return array
+     */
+    public function fixDate(array $lines, $separator = ',', $enclosure = '"')
```

Comment:
'fixDate' is not really a good name, what is it fixing? 

Basically it's going from us to europe style, maybe we can improve the name to explain what it's doing.